src/components/DataTable/index.tsx

```diff
@@ -424,7 +426,7 @@ export const DataTable = <T extends AnyObject>({
     if (!displayData.length) {
       const emptyMessage = hasAppliedFilters
         ? "There are no items matching selected filters"
-        : "There are no contract storage items";
+        : defaultEmptyMessage || "There are no contract storage items";
```

Comment:
It doesn't look like we're using `emptyMessage` prop anywhere.

```suggestion
        : emptyMessage || "There are no contract storage items";
```

Reply:
I'm renaming the prop to `defaultEmptyMessage`, so the external name is `emptyMessage`. See https://github.com/stellar/laboratory/pull/1330/files#diff-d85b64b87b8bec7dfac2aa55e7f7db25e3273806964c25be26954e249c862dfeR72